app/build.gradle

```diff
@@ -1,11 +1,15 @@
-apply plugin: 'com.android.application'
+plugins {
+    alias libs.plugins.android.application
+    alias libs.plugins.kotlin.android
+}
 
 android {
-    compileSdkVersion 34
+    namespace "io.kommunicate.app"
     defaultConfig {
         applicationId "io.kommunicate.app"
-        minSdkVersion 17
```

Comment:
Sure, This is for the sample app only not for SDK as one the library which I integrated requires min SDK 19.

Name | Version Code | Percentage Supported
**Android 4.4 (KitKat)** | 19 | 100.0%
**Android 4.4 w (Kitkat)** | 20 | 100.0% ~ (As it is not mentioned clearly in the [composables](https://www.composables.com/tools/distribution-chart) website. This Android update adds the support for wearOS as well. So there wont be any significant change in Supporting devices. 

Also, I think we should migrate our SDK to min 20 as well because it will cover almost all the android devices. Also, It is also required by most of the libraries once we upgrade them or if we add the new library in our project.